yarn.lock

```diff
@@ -2,12 +2,843 @@
 # yarn lockfile v1
```

Comment:
Lock files are infeasible to review, but can introduce *anything* into the repo :), so I'll regenerate it myself from the fresh package.json.

Reply:
Definitely the safe bet, given some of the upstream attacks that have occurred in the past.